README.md

```diff
@@ -16,19 +16,22 @@ The documentation in this repository is comprised of sequential chapters which i
 Most API operations rely on the context of a site. Whether you’ve already got a site created in the Sitelink3D v2 web portal or you’re relying entirely on the API, this chapter will introduce site basics and point to code examples that interact with ```siteowner```, the microservice responsible for site creation and management.
 
 ### 2. RDM (Replicated Domain Model)
-Don’t let the terminology put you off. RDM is simply the technology we’ve built for defining and distributing objects within the Sitelink3D v2 ecosystem. RDM is one of the simplest ways to get working with our API so we’ll be looking at examples of creating objects by writing to RDM and then using views to read data back.
+Don’t let the terminology put you off. RDM is simply the technology we’ve built for defining and distributing objects within the Sitelink3D v2 ecosystem. RDM is one of the simplest ways to get working with our API. In this chapter we’ll be looking at examples of creating objects by writing to the ```rdm_log``` microservice and then using views with the ```rdm``` microservice to read data back.
```

Comment:
rdm_log isn't the microservice it's just part of the route

Reply:
Thanks, have fixed, rebased, pushed.